index.js

```diff
@@ -40,6 +42,24 @@ function increaseSpecifityOfRule(rule, opts) {
 	}
 }
 
+function isDisabled(node, status) {
+	if (!node) return false;
+
+	var prev = node.prev();
+
+	if (prev && prev.type === 'comment') {
+		if (BLOCK_DISABLE.test(prev.text)) {
+			status.disabled = true;
+			return true;
+		} else if (BLOCK_ENABLE.test(prev.text)) {
+			status.disabled = false;
+			return false;
+		}
+	}
+
+	// if no value, use last status.
+	return status.disabled;
+}
```

Comment:
Currently this fails with the following example (let's make sure to add a test),

```css
/* postcss-increase-specificity disable */
/* Some other comment you picked up instead of the one above */
+.foo {
	background: #f00;
}
/* postcss-increase-specificity enable */
```

I'm also concerned with other random nodes in between. We should check between the current rule node and the last rule node we walked, `checkForEnableDisableComments(fromNode, toNode)`